src/odemis/driver/pigcs.py

```diff
@@ -511,7 +511,7 @@ def recoverTimeout(self):
         try:
             for i in range(2):
                 self.busacc.flushInput()
-                resp = self.busacc.sendQueryCommand(self.address, "ERR?\n")
+                resp = self.busacc.sendQueryCommand(self.address, "ERR?\n").decode('latin1', 'backslashreplace')
```

Comment:
Don't decode to a string using "backslashreplace". That's just useful when outputting logs, to make it readable. However, for data processing, you never want to have this.

Reply:
What about functions like `GetIdentification` or `GetStageName`? Should I use backslashreplace there or not?

Reply:
Normally, for the wrapping functions, you shouldn't escape the returned data. Either detect there is something wrong in it, or pass it as-is (just do that, it's easier ;-) ).